mqtt_client_interfaces/srv/NewRos2ToMqtt.srv

```diff
@@ -0,0 +1,11 @@
+# This service is used to dynamically register new ROS2 -> MQTT mappings. 
+# Syntax is the same as the ones used on config files.
+# Remember to also register MQTT -> ROS2.

```

Comment:
In many of our own use cases, you only want to have uni-directional bridges on a single host, e.g., if sending ROS data via MQTT to another host.

```suggestion
# This service is used to dynamically register new ROS -> MQTT mappings.

```